components/api/db/cellery-hub-db.bal

```diff
@@ -211,16 +211,69 @@ returns json | error {
         table<gen:OrgListResponseAtom> resOrgListAtom = check connection->select(SEARCH_USER_ORGS_QUERY, gen:OrgListResponseAtom, orgName, userId,
         resultLimit, offset, loadToMemory = true);        
         foreach int i in 0...resOrgListAtom.count()-1{
-            gen:OrgListResponseAtom olra = check gen:OrgListResponseAtom.convert(resOrgListAtom.getNext());
-            olr.data[i] = olra;
-            if (imageCountMap[olra.orgName] is ()){
-                imageCountMap[olra.orgName] = 0;
+            gen:OrgListResponseAtom orgListResponseAtom = check gen:OrgListResponseAtom.convert(resOrgListAtom.getNext());
+            orgListResponse.data[i] = orgListResponseAtom;
+            if (imageCountMap[orgListResponseAtom.orgName] is ()){
+                imageCountMap[orgListResponseAtom.orgName] = 0;
             }
-            olr.data[i]["imageCount"] = <int>imageCountMap[olra.orgName];
+            orgListResponse.data[i]["imageCount"] = <int>imageCountMap[orgListResponseAtom.orgName];
         }
         resOrgListAtom.close();
     } else {
         log:printDebug(io:sprintf("No organization found for userId %s, with the orgName \'%s\'",userId, orgName));
     }
-    return check json.convert(olr);
+    return check json.convert(orgListResponse);
+}
+
+public function getPublicImagesOfanORG(string orgName, string imageName, string orderBy, int offset, int resultLimit)
```

Comment:
Please refactor this method name according to naming conventions.

Suggestion - getPublicImagesOfOrg()

Rename the other method as well